app/model.go

```diff
@@ -22,6 +22,7 @@ type AuthenticationReport struct {
 	Status    string `json:"status"`
 	ErrorText string `json:"error_text"`
 	ErrorCode string `json:"error_code"`
+	Errors    string `json:"errors,omitempty"` // Redundant and deprecated field to maintain compatibility with FHX.
```

Comment:
I would change the comment to:
`Deprecated: ...` that way the IDE is also striking the prop too.